framework/include/controls/RealFunctionControl.h

```diff
@@ -62,7 +62,7 @@ class RealFunctionControl : public Control
   Function & _function;
 
   /// Vector of parameters to change
-  std::vector<Real *> _parameters;
+  ControllableParameter<Real> _parameters;
```

Comment:
It's weird to get these "by copy"... it doesn't match the way we do anything else in MOOSE